docker/node/bin/run-deploy-storefront

```diff
@@ -115,6 +115,13 @@ isCurrentBranchMainBranch() {
   return 1
 }
 
+isCurrentBranchMainOrVBranch() {
+  if [[ $(isCurrentBranchMainBranch) || "${P_CURRENT_BRANCH}" == v* ]]; then
```

Comment:
`"${P_CURRENT_BRANCH}" == v*` is this comparison really working?

Reply:
yes
```
P_CURRENT_BRANCH=v1  
➜  components-js git:(issue/1988) ✗ if [[ "${P_CURRENT_BRANCH}" == v* ]]; then echo 'yo'; fi
yo
➜  components-js git:(issue/1988) ✗ P_CURRENT_BRANCH=issue/asd                              
➜  components-js git:(issue/1988) ✗ if [[ "${P_CURRENT_BRANCH}" == v* ]]; then echo 'yo'; fi
```